<?xml version="1.0"?>
<launch>
	
	<param name="/use_sim_time" value="false"/>
	<arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
	<arg name="robot_env" default="$(optenv ROBOT_ENV !!NO_ROBOT_ENV_SET!!)"/>
	<arg name="pkg_env_config" default="$(find cob_default_env_config)"/>

	<!-- start cob_frame_tracker_action_client -->
	
	<node ns="/torso" pkg="cob_frame_tracker" type="cob_frame_tracker_action_client_node" name="cob_frame_tracker_action_client" respawn="false" output="screen"/>
	
</launch>
